ml-completion/src/main/kotlin/org/rust/ml/RsMLRankingProvider.kt

```diff
@@ -6,10 +6,15 @@
 package org.rust.ml
 
 import com.intellij.internal.ml.catboost.CatBoostJarCompletionModelProvider
+import com.intellij.internal.ml.completion.DecoratingItemsPolicy
 import com.intellij.lang.Language
 import org.rust.lang.RsLanguage
 
 @Suppress("UnstableApiUsage")
 class RsMLRankingProvider : CatBoostJarCompletionModelProvider("Rust", "rust_features", "rust_model") {
     override fun isLanguageSupported(language: Language): Boolean = language == RsLanguage
+    override fun getDecoratingPolicy(): DecoratingItemsPolicy = DecoratingItemsPolicy.Composite(
```

Comment:
Looks like this API is only available since 211

Reply:
Thank you, fixed!